main.tf

```diff
@@ -0,0 +1,27 @@
+# Configure AWS
+provider "aws" {
```

Comment:
I don't think this block is necessary.

Reply:
It turns out that if you remove that block, you get hit with this prompt, so I'm going to put it back in:
```
❱ terraform apply
provider.aws.region
  The region where AWS operations will take place. Examples
  are us-east-1, us-west-2, etc.

  Default: us-east-1
  Enter a value: 
```

Reply:
You have to remove that block as well as the corresponding block in `variables.tf`.

Reply:
Actually that doesn't work.  Let me look into this.

Reply:
My latest push resolves this.

Reply:
💪 💼 @jsf9k